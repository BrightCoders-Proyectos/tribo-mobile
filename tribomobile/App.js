/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow strict-local
 */

import React from 'react';
import {SafeAreaView, Text, StatusBar, Modal} from 'react-native';
import titlesText from './src/TitlesText';
import ContentText from './screensText/ContentText';
import LogoTribo from './components/LogoTribo';
import ExampleButtons from './ExampleButtons';
import MainScreen from './components/MainScreen';
import ModalInfoStore from './components/modals/ModalInfoStore';
import LittlePinInfo from './components/modals/LittlePinInfo';
import SideNavScreen from './screens/SideNavScreen';
import RegisterMarket  from './screens/RegisterMarket';
import RegistrationScreen  from './screens/RegistrationScreen';
import WelcomeScreen from './components/screens/welcome/WelcomeScreen';
import LoadingScreen from './components/screens/welcome/LoadingScreen';
import LoginScreen from './screens/Login';
import EditAccountScreen from './screens/profile/EditAccountScreen';

import {
  ModalDeleteStore,
  ModalDeleteStoreTexts,
} from './components/modals/ModalDeleteStore';

const App: () => React$Node = () => {
  const userDummy = {
    name: 'Daniel Peña Sanchez',
    email: 'daniel.peña@gmail.com',
  };
  return (
    <>
      <StatusBar barStyle="dark-content" />
      <SafeAreaView>
        {/* <RegistrationScreen/> */}
        <EditAccountScreen userAccount={userDummy}/>
      </SafeAreaView>
    </>
  );
};

export default App;
