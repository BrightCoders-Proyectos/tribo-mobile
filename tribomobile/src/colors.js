const colors = {
  BlueDark: '#1F1F1F',
  BlueLight: '#12a4ed',
  OrangeFonts: '#e98d1e',
  BlueP: '#037d94',
  GreenContent: '#7ad520',
  GrayLight: '#939393',
  GrayDark: '#4A4A4A',
  Mediumvioletred: '#c71585',
  OrangeDark: '#fe8500',
  Yellow: '#FFCA00',
  White: '#ffffff',
};
export default colors;
