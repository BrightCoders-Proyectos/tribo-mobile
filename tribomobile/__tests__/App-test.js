/**
 * @format
 */
import 'react-native';
import React from 'react';
import App from '../App';
import colors from '../src/Colors';
import renderer from 'react-test-renderer';
import {
  ModalDeleteStore,
  ModalDeleteStoreTexts,
} from '../components/modals/ModalDeleteStore';

const FixedDimensionsBasics = () => {
  return (
    <View>
      <View
        style={{width: 50, height: 50, backgroundColor: colors.GreenContent}}
      />
      <View
        style={{
          width: 100,
          height: 100,
          backgroundColor: colors.mediumvioletred,
        }}
      />
      <View style={{width: 150, height: 150, backgroundColor: colors.blueP}} />
    </View>
  );
};

it('renders correctly modal to delete a store', () => {
  const fields = {
    title: ModalDeleteStoreTexts.title.business,
    description: 'Estas seguro de eliminar la "fonda de do√±a luisa?"',
  };
  const tree = renderer.create(<ModalDeleteStore {...fields} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly modal to delete an account', () => {
  const fields = {
    title: ModalDeleteStoreTexts.title.account,
    description: ModalDeleteStoreTexts.description.account,
  };
  const tree = renderer.create(<ModalDeleteStore {...fields} />).toJSON();
  expect(tree).toMatchSnapshot();
});
