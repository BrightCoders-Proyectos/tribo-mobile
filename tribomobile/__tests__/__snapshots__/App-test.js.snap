// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly modal to delete a store 1`] = `
Array [
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(10,9,9,0.7)",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Modal
      animationType="slide"
      hardwareAccelerated={false}
      onRequestClose={[Function]}
      transparent={true}
      visible={false}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(10,9,9,0.7)",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#f7f4f4",
              "borderRadius": 7,
              "elevation": 5,
              "margin": 20,
              "maxWidth": "92%",
              "paddingBottom": 40,
              "paddingTop": 25,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.25,
              "shadowRadius": 3.84,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-end",
                "marginHorizontal": 20,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../../assets/close.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "tintColor": "#828894",
                    "width": 20,
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "marginHorizontal": 15,
                "maxWidth": "98%",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "black",
                  "fontSize": 30,
                  "fontWeight": "bold",
                  "textAlign": "center",
                }
              }
            >
              Eliminar tu negocio
            </Text>
            <Text
              style={
                Object {
                  "color": "black",
                  "fontSize": 24,
                  "textAlign": "center",
                }
              }
            >
              Estas seguro de eliminar la "fonda de do√±a luisa?"
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
                "paddingHorizontal": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                }
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#037D94",
                    "borderColor": "#037D94",
                    "borderRadius": 50,
                    "borderWidth": 2,
                    "flexDirection": "row",
                    "height": 50,
                    "justifyContent": "center",
                    "marginTop": 40,
                    "opacity": 1,
                    "width": 165,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "fontWeight": "bold",
                        "paddingHorizontal": 10,
                      },
                      Object {
                        "color": "#FFFFFF",
                        "fontSize": 20,
                      },
                    ]
                  }
                >
                  Confirmar
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                }
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#9CA3AF",
                    "borderColor": "#9CA3AF",
                    "borderRadius": 50,
                    "borderWidth": 2,
                    "flexDirection": "row",
                    "height": 50,
                    "justifyContent": "center",
                    "marginTop": 40,
                    "opacity": 1,
                    "width": 165,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "fontWeight": "bold",
                        "paddingHorizontal": 10,
                      },
                      Object {
                        "color": "#FFFFFF",
                        "fontSize": 20,
                      },
                    ]
                  }
                >
                  Cancelar
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </Modal>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#F194FF",
        "opacity": 1,
        "padding": 10,
        "width": "80%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontWeight": "bold",
          "textAlign": "center",
        }
      }
    >
      Show Modal delete store
    </Text>
  </View>,
]
`;

exports[`renders correctly modal to delete an account 1`] = `
Array [
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(10,9,9,0.7)",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Modal
      animationType="slide"
      hardwareAccelerated={false}
      onRequestClose={[Function]}
      transparent={true}
      visible={false}
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "rgba(10,9,9,0.7)",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#f7f4f4",
              "borderRadius": 7,
              "elevation": 5,
              "margin": 20,
              "maxWidth": "92%",
              "paddingBottom": 40,
              "paddingTop": 25,
              "shadowColor": "#000",
              "shadowOffset": Object {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.25,
              "shadowRadius": 3.84,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-end",
                "marginHorizontal": 20,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../../assets/close.png",
                  }
                }
                style={
                  Object {
                    "height": 20,
                    "tintColor": "#828894",
                    "width": 20,
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "marginHorizontal": 15,
                "maxWidth": "98%",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "black",
                  "fontSize": 30,
                  "fontWeight": "bold",
                  "textAlign": "center",
                }
              }
            >
              Eliminar tu cuenta
            </Text>
            <Text
              style={
                Object {
                  "color": "black",
                  "fontSize": 24,
                  "textAlign": "center",
                }
              }
            >
              ¬øEstas seguro que quieres eliminar tu cuenta?
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
                "paddingHorizontal": 10,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                }
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#037D94",
                    "borderColor": "#037D94",
                    "borderRadius": 50,
                    "borderWidth": 2,
                    "flexDirection": "row",
                    "height": 50,
                    "justifyContent": "center",
                    "marginTop": 40,
                    "opacity": 1,
                    "width": 165,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "fontWeight": "bold",
                        "paddingHorizontal": 10,
                      },
                      Object {
                        "color": "#FFFFFF",
                        "fontSize": 20,
                      },
                    ]
                  }
                >
                  Confirmar
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                }
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#9CA3AF",
                    "borderColor": "#9CA3AF",
                    "borderRadius": 50,
                    "borderWidth": 2,
                    "flexDirection": "row",
                    "height": 50,
                    "justifyContent": "center",
                    "marginTop": 40,
                    "opacity": 1,
                    "width": 165,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "fontWeight": "bold",
                        "paddingHorizontal": 10,
                      },
                      Object {
                        "color": "#FFFFFF",
                        "fontSize": 20,
                      },
                    ]
                  }
                >
                  Cancelar
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </Modal>
  </View>,
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#F194FF",
        "opacity": 1,
        "padding": 10,
        "width": "80%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontWeight": "bold",
          "textAlign": "center",
        }
      }
    >
      Show Modal delete store
    </Text>
  </View>,
]
`;
